askforhelp.rst

```diff
@@ -63,17 +63,16 @@ The big picture - An opening sentence should state the general problem that you
 
 **A snapshot of your environment** - For Plone and for other relevant products: provide version numbers. e.g., "I'm running Plone 5.0.4 under Python 2.7.11."
 
-**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer. The expected result .
+**Steps to reproduce the issue** - Give information about your ideas of how this error appeared, what caused it or anything that could lead to reproducing the error on another computer, including your buildout.cfg, versions.cfg, version numbers of installed add-ons, detailed command lines, the complete error message stack. Mention your expected result.
 
-Asking help in real time chat (IRC)
------------------------------------
+Asking for help in online chat
+------------------------------
 
-Plone community real time chat is at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
+The Plone community uses `online chat <https://plone.org/support/chat>`_, specifically `Gitter <https://gitter.im/plone/public>`_ and traditional Plone IRC chat at the  `#plone <irc://irc.freenode.net/#plone>`_ IRC channel on the freenode IRC network.
 
```

Comment:
I agree and understand. We went through this discussion and ended up deciding to leave Slack fans on their own and able to use Slack (they continue to do so), but for official chat we leave IRC for fans of that and we introduce Gitter because it's friendly to open source projects, doesn't have the problems Slack has caused for open source projects, and it authenticates with GitHub, which makes that match plone.org and our code repos and issue trackers. Also, Gitter's requirement for authentication lets us respond to questions & notify people after they'd normally have disconnected from IRC (something that drives me bananas not just personally but also from the community perception perspective).

In other words, we preserve choice and freedom but steer newcomers to Gitter if they don't like or find IRC unapproachable (which we've heard a lot of).